### ISSUE TITLE
[Stale Documentation] VoidMonster custom drops (Missing key 'type')

### ISSUE THREAD

**Issue by voidentente** [ReikaKalseki/Reika_Mods_Issues#3059]
**Mod: VoidMonster 1.7.10 V31a & DragonAPI 1.7.10 V31b
TL;DR: VoidMonster fails post-initialization after adding custom drop, throwing "Missing key 'type'" error.
DragonAPI did not crash.
Similar to #2772 (?)**

I am trying myself at LuaBlock syntax, but am running into a reoccurring error:

`[15:28:40] [Client thread/ERROR]: Caught exception from VoidMonster
java.lang.IllegalArgumentException: Missing key 'type' for '64846f4'
	at Reika.DragonAPI.Instantiable.IO.LuaBlock.getString(LuaBlock.java:129) ~[LuaBlock.class:?]
	at Reika.DragonAPI.Instantiable.IO.CustomRecipeList.parseLuaBlocks(CustomRecipeList.java:162) ~[CustomRecipeList.class:?]
	at Reika.DragonAPI.Instantiable.IO.CustomRecipeList.load(CustomRecipeList.java:145) ~[CustomRecipeList.class:?]
	at Reika.DragonAPI.Instantiable.IO.CustomRecipeList.load(CustomRecipeList.java:103) ~[CustomRecipeList.class:?]
	at Reika.VoidMonster.Auxiliary.VoidMonsterDrops.loadCustomDrops(VoidMonsterDrops.java:61) ~[VoidMonsterDrops.class:?]
	at Reika.VoidMonster.VoidMonster.postload(VoidMonster.java:196) ~[VoidMonster.class:?]`

The config used is this:
(config/Reika/Void Monster_CustomDrops/custom.drops)

`{
	anomalyDrop = {
		item = "mo:gravitational_anomaly"
		max = 1
		min = 1
	}
}`

I tried the obvious, however adding a key-value pair like `type = "..."` or `"type" = "..."` does not fix the error.
Removing the above file does.
Regardless, I should note that the examples in config/Reika/Void Monster_CustomDrops/example.drops thankfully throw the same error, so this isn't just some cursed file.

Bug or outdated documentation?

**Comment by voidentente** [comment]
Ah. The following syntax works, but it does differ from the given documentation:

`{
	type = "Gravitational Anomaly"
    item = "mo:gravitational_anomaly"
	max = 1
	min = 1
}`
